src/widgets/HazardPoint.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
not sure whats up with the diff - looks like I messed up the order of things. This has already had a PR and been merged :(